vendor/vendor-android/src/main/kotlin/com/malinskiy/marathon/android/extension/ConfigurationExtensions.kt

```diff
@@ -36,7 +37,7 @@ fun VideoConfiguration.toScreenRecorderCommand(remoteFilePath: String): String {
     if (timeLimit > 0) {
         sb.append("--time-limit ")
         var seconds = TimeUnit.SECONDS.convert(timeLimit, timeLimitUnits)
-        if (seconds > 180) {
+        if (seconds > 180 && ((device?.apiLevel ?: 0) < 34 || !increasedTimeLimitFeatureEnabled)) {
```

Comment:
@Vacxe  But if (seconds > 180 && (device?.apiLevel ?: 0) >= 34 && increasedTimeLimitFeatureEnabled) == true -> we don't need to trim video, we need to do it if:
1) (device?.apiLevel ?: 0) < 34 AND seconds > 180 (even we have enabled feature -> we can't create > 180 sec videos for devices with apiLevel < 34) OR
2) our feature is disabled AND seconds > 180 (it doesn't matter what the api version - I want to save the old behavior in case of disabled feature)